CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+- bin/check-es-cluster-health.rb and bin/check-es-cluster-status.rb fixed --alert-status failing to alert and allow absent value to alert on any status
```

Comment:
This should go under a `### Fixed` sub header per: https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md